src/components/RecipeList.js

```diff
@@ -0,0 +1,26 @@
+require('normalize.css/normalize.css');
+require('styles/App.css');
+
+import React from 'react';
+
+var RecipeList = React.createClass({
+  render: function() {
+    return (
+      <ul className="recipe-list">
+        <li>
+          <ul className="recipe-list">
+            {
+              this.props.items.map(item => {
```

Comment:
the good practice is to assign properties available in the `this.props` object to constant before use it in JSX, so it would look like below (at the beginning of the `render` method):

```
const { items } = this.props;
```

and then use it like below: 

```
items.map(item => {
  ...
}
```